doc/user_guide/region_of_interest.rst

```diff
@@ -36,8 +36,8 @@ added before calling :py:meth:`~.roi.BaseInteractiveROI.interactive`.
 
 .. code-block:: python
 
-    >>> import scipy.misc
-    >>> im = hs.signals.Signal2D(scipy.misc.ascent())
+    >>> import scipy.datasets
```

Comment:
Simplify to the following here in other places as elsewhere, `scipy.ndimage` is also used.
```suggestion
    >>> import scipy
```